lib/attachment_to_html/adapters/google_docs_viewer.rb

```diff
@@ -23,11 +23,7 @@ def initialize(attachment, opts = {})
       private
 
       def parse_body
-        %Q(<iframe src="#{ protocol }://docs.google.com/viewer?url=#{ attachment_url }&amp;embedded=true" width="100%" height="100%" style="border: none;"></iframe>)
-      end
-
-      def protocol
-        AlaveteliConfiguration.force_ssl ? 'https' : 'http'
+        %Q(<iframe src="https://docs.google.com/viewer?url=#{ attachment_url }&amp;embedded=true" width="100%" height="100%" style="border: none;"></iframe>)
```

Comment:
I don't disagree, but _why_? Commit message should explain.

Reply:
I have updated the comment